db/structure.sql

```diff
@@ -9,20 +9,6 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
---
```

Comment:
unnecessary structure changes here -- lets pair on fixing this this afternoon...